routes/web.php

```diff
@@ -24,6 +24,7 @@
 
 Route::get('/settings/', 'SettingsController@index')->name('settings');
 Route::post('/settings/', 'SettingsController@save');
+Route::post('/settings/connections/telegram/delete', 'SettingsController@deleteTelegramConnection')->name('settings.connections.telegram.delete');
```

Comment:
Die Zeile sieht etwas lang aus. Ich würde `->name(...)` in eine neue Zeile setzen.